modules/core/include/object.h

```diff
@@ -2,39 +2,37 @@
  * @file object.h
  * @author WARG
  *
+ * @brief Container class for storing information about
```

Comment:
This describes a class not a file. I'm not really sure why this needed to be moved.